tests/collection.js

```diff
@@ -157,7 +150,7 @@ describe( 'Collection', () => {
 
 				collection.add( item );
 
-				expect( item ).to.have.property( 'id', '3' );
+				expect( item.id ).to.be.above( '3' );
```

Comment:
I'm afraid that this will always be true, because since uid()'s return value is cached and increased every time the function is used, the latest id can be far more than 3.

I'll try to figure out what this test checks precisely and get back to you.